modules/core/src/main/java/org/apache/ignite/internal/thread/StripedThreadPoolExecutor.java

```diff
@@ -105,7 +106,7 @@ public void execute(Runnable cmd) {
      * @throws NullPointerException       if the task is {@code null}.
      */
     public CompletableFuture<?> submit(Runnable task, int idx) {
-        return CompletableFuture.runAsync(task, execs[threadId(idx)]);
+        return CompletableFuture.runAsync(task, commandExecutor(idx));
```

Comment:
`@param idx` is missed in javadoc.
Would you please fix this?

Reply:
Done.